Libraries/Utilities/Appearance.js

```diff
@@ -85,6 +85,22 @@ module.exports = {
     return nativeColorScheme;
   },
 
+  setColorScheme(colorScheme: ?ColorSchemeName): void {
+    const nativeColorScheme: ?string =
```

Comment:
```suggestion
    const nativeColorScheme =
```

Flow should be able to infer this to an always non-null string, now that we set a value in the nullish case.